### PR TITLE
🐛 fix run_tests.sh

### PR DESCRIPTION
**Pull request by Kanti**

### CODE REVIEW

File: run_tests.sh [hunk 1]
```diff
@@ -14,8 +14,10 @@ test "$(php -m | grep xdebug)" = "" &&
 test "$(php -i | grep auto_prepend_file)" = "auto_prepend_file => /opt/docker/profiler.php => /opt/docker/profiler.php" &&
 # test tideways is installed
 test "$(php -m | grep tideways)" = "tideways_xhprof" &&
-# is hirak/prestissimo installed in composer
-[[ "$(composer global show 2>&1 | grep 'hirak/prestissimo')" =~ ^hirak/prestissimo.* ]] &&
+# is hirak/prestissimo installed in composer1
+[[ "$(composer1 global show 2>&1 | grep 'hirak/prestissimo')" =~ ^hirak/prestissimo.* ]] &&
+# is hirak/prestissimo installed in composer1
```
Comment (Kanti) [inline run_tests.sh:19]:
```suggestion
# is hirak/prestissimo installed in composer2
```

File: run_tests.sh [hunk 2]
```diff
@@ -35,7 +37,9 @@ ping -V &&
 # docker should be installed
 docker --version &&
 # composer should be installed
```
Comment (Kanti) [inline run_tests.sh:39]:
```suggestion
# composer1 should be installed
```

File: run_tests.sh [hunk 3]
```diff
@@ -35,7 +37,9 @@ ping -V &&
...
-composer --version &&
+composer1 --version &&
+# composer should be installed
```
Comment (Kanti) [inline run_tests.sh:41]:
```suggestion
# composer2 should be installed
```

File: run_tests.sh [hunk 4]
```diff
@@ -34,8 +36,10 @@ tree --version &&
 ping -V &&
 # docker should be installed
 docker --version &&
-# composer should be installed
-composer --version &&
```
Comment (ChrisB9) [inline run_tests.sh:38]: composer without providing an env should work as well
same output as composer 2.0

**Review by ChrisB9 — CHANGES_REQUESTED** [review]

File: run_tests.sh [hunk 5]
```diff
@@ -34,8 +36,10 @@ tree --version &&
...
+# composer1 should be installed
+composer1 --version &&
```
Comment (Kanti) [inline run_tests.sh:40]:
```suggestion
# composer should be installed (version 2)
composer --version &&
# composer1 should be installed
composer1 --version &&
```

**Review by ChrisB9 — APPROVED** [review]

**Pull request merged**